packages/node-core/src/indexer/testing.service.ts

```diff
@@ -119,7 +119,7 @@ export abstract class TestingService<B, DS> {
 
   private async runTest(test: SubqlTest, sandbox: TestSandbox) {
     logger.info(`Starting test: ${test.name}`);
-    const schema = `test-${this.nodeConfig.subqueryName}`;
+    const schema = `test-${this.nodeConfig.dbSchema}`;
```

Comment:
Won't this now end up being `test-test-*` because node config will update the db schema?